### PR TITLE
feature: Combined relation managers and form

### PR DESCRIPTION
**Pull request by danharrin**
Screenshot 2022-08-23 at 09 14 46

### CODE REVIEW

File: packages/admin/resources/views/resources/pages/edit-record.blade.php [hunk 1]
```diff
@@ -6,18 +6,42 @@
...
 
-    @if (count($relationManagers = $this->getRelationManagers()))
-        <x-filament::hr />
+    @php
+        $relationManagers = $this->getRelationManagers();
+    @endphp
 
-        <x-filament::resources.relation-managers :active-manager="$activeRelationManager" :managers="$relationManagers" :owner-record="$record" :page-class="static::class" />
+    @if ((! $this->hasCombinedRelationManagersWithForm()) || (! count($relationManagers)))
+        {{ $form() }}
```
Comment (zepfietje) [inline packages/admin/resources/views/resources/pages/edit-record.blade.php:25]:
```suggestion
        {{ $form }}
```

File: packages/admin/resources/views/resources/pages/edit-record.blade.php [hunk 2]
```diff
@@ -6,18 +6,42 @@
...
+        <x-filament::resources.relation-managers
+            :active-manager="$activeRelationManager"
+            :form-tab-label="$this->getFormTabLabel()"
+            :managers="$relationManagers"
+            :owner-record="$record"
+            :page-class="static::class"
+        >
+            @if ($this->hasCombinedRelationManagersWithForm())
+                <x-slot name="form">
+                    {{ $form() }}
```
Comment (zepfietje) [inline packages/admin/resources/views/resources/pages/edit-record.blade.php:42]:
```suggestion
                    {{ $form }}
```

**Review by zepfietje — CHANGES_REQUESTED** [review]
Wouldn't this work?

File: packages/admin/resources/views/resources/pages/edit-record.blade.php (hunk 1 above)
Comment (danharrin) [reply packages/admin/resources/views/resources/pages/edit-record.blade.php:25]: No, this is the syntax for the Blade @capture directive

Comment (zepfietje) [reply packages/admin/resources/views/resources/pages/edit-record.blade.php:25]: I'm sorry, I misread the package docs.

**Comment by cheesegrits** [comment]
w00t!  I was literally just now about to head to Discord to ask if this would be possible as a feature, and happened to be scanning thru PR's to see if mine had passed checks.

**Comment by joshlowry** [comment]
> w00t! I was literally just now about to head to Discord to ask if this would be possible as a feature, and happened to be scanning thru PR's to see if mine had passed checks.

Ditto! What fantastic timing. Thanks @danharrin!

**Pull request merged**